src/gt4py/backend/dawn_backends.py

```diff
@@ -506,6 +506,7 @@ def generate(
             os.makedirs(os.path.dirname(file_name), exist_ok=True)
             with open(file_name, "w") as file:
                 file.write(module_source)
+            cls.update_cache(stencil_id, dict(pyext_file_path=pyext_file_path))
```

Comment:
Due to #134 this line will have to be changed everything else should stay the same after the merge.

```suggestion
            self.builder.caching.update_cache_info(stencil_id, dict(pyext_file_path=pyext_file_path))
```